scripts/release.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+# should be run as:
+# ./release.sh x.x.x start|end description
+set -eo pipefail
+
+USAGE="Usage:
+${0} x.x.x start|end description, E.g. to start a release run:
```

Comment:
@lucafavatella is start/finish more appropriate wording ?

Reply:
changed in 865ce23